### ISSUE TITLE
Convert percy script into percy snapshot calls

### ISSUE THREAD

**Issue by blms** [Princeton-CDH/geniza#487]
From https://github.com/Princeton-CDH/geniza/pull/484#discussion_r778329741:
> I noticed in the percy docs that [percy script](https://docs.percy.io/docs/percyscript) has been deprecated and we should switch to using the [snapshot command](https://docs.percy.io/docs/cli-snapshot) — we don't need to change it now necessarily, but you are making a lot of changes and I wonder if it would simplify anything here.

This issue tracks the update to the snapshot CLI.

**Comment by kseniaryzhova** [comment]
@blms is this still relevant?

**Comment by blms** [comment]
@kseniaryzhova Yes, we still haven't done this upgrade, but it's not a critical one. I think the Icebox makes sense for this—good to keep on hand in case we run into problems with Percy.

**Comment by mabdellatif88** [comment]
Removed to simplify dependencies
